stores/multipart.go

```diff
@@ -29,11 +29,13 @@ type (
 
 	dbMultipartPart struct {
 		Model
-		Etag                string `gorm:"index"`
-		PartNumber          int    `gorm:"index"`
+		PartNumber          int `gorm:"index"`
 		Size                uint64
 		DBMultipartUploadID uint      `gorm:"index;NOT NULL"`
 		Slabs               []dbSlice `gorm:"constraint:OnDelete:CASCADE"` // CASCADE to delete slices too
+
+		Etag     string `gorm:"index"`
+		MimeType string `gorm:"index"`
```

Comment:
Why does a part need a MimeType? Shouldn't that be stored on the upload?
Seems like S3 uses the `Content-Type` request header to determine the mime type in `PutObject` and `CreateMultipartUpload`.
So maybe it's fine if we do the same and expect the client to determine the mime type?

Reply:
I wasn't too sure about this but I figured I wasn't doing anything wrong with doing it like this. You are essentially left with the mime type you would get (likely) if you would let the client determine the mime type or if you would upload the object in one go. At the cost of determining the type for every part and storing it on every part, which I figured was fine..

I'm ok to remove this though, I had similar thoughts/feelings.

Reply:
Some context on why we need to have a mime type is that we need to avoid serve content sniffing for it because that will complicate the `ReadSeeker`. It's not that annoying but at that point in time we'll try and sniff it anyway so we might as well do that on upload.